src/components/Single-Article.jsx

```diff
@@ -2,18 +2,20 @@ import { useEffect } from "react"
 import { useState } from "react"
 import { useParams } from "react-router-dom"
 import { getSingleArticle, getCommentsForArticle } from "../../api.get"
+import { patchVotes } from "../../api.patch"
 
 export const SingleArticle = () => {
```

Comment:
i currently cant navigate to the individual articles as there is no link from the articles page!